explorer/client/src/utils/static/movecall-tx-response.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
This JSON file is in the correct directory; although `findDataFromID` will need to tap into this JSON file also.